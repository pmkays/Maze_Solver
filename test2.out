=========S*=========
==========*=========
==========*=========
==========***=======
============*=======
==========***=======
==========*=========
===***====**========
===*=*=====*========
===*=*******========
===*================
===*******==========
=========*****======
=============*==E===
=============*==*===
=============****===
====================
====================
====================
====================
East
South
South
South
East
East
South
South
West
West
South
South
East
South
South
West
West
West
West
West
West
North
North
West
West
South
South
South
South
East
East
East
East
East
East
South
East
East
East
East
South
South
South
East
East
East
North
North
