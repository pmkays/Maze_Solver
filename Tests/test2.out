=========S*=========
==========*=========
==========*=========
==========***=======
============*=======
==========***=======
==========*=========
===***====**========
===*=*=====*========
===*=*******========
===*================
===*******==========
=========*****======
=============*==E===
=============*==*===
=============****===
====================
====================
====================
====================
east
south
south
south
east
east
south
south
west
west
south
south
east
south
south
west
west
west
west
west
west
north
north
west
west
south
south
south
south
east
east
east
east
east
east
south
east
east
east
east
south
south
south
east
east
east
north
north
